scripts/garage

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+# pylint: skip-file
```

Comment:
why? it can be linted like anything else.

Reply:
pylint seems to be buggy because of the name of this file (`garage`). But anyway I will replace `skip-file` with specific disables.